services/_internal/maven-EGI/workflowEGI/src/org/json/JSONWriter.java

```diff
@@ -0,0 +1,326 @@
+package org.json;

```

Comment:
You put the entire JSON library in your code. This is why your pull request is so big. Maven should handle all the dependencies. Please remove all those libraries related to JSON (.../src/org/json/....) and all other 3rd party libraries (in case I did not find them) from your pull request. Maven should fetch them, but not you